.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "wasmCloud",
+    "image": "rust:bullseye",
+    "features": {
+        "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-binstall:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-watch:latest": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-nextest:latest": {},
```

Comment:
```suggestion
```

Let's start with just minimal toolset required to develop

Reply:
removed